README.md

```diff
@@ -71,6 +71,46 @@ testAgent.on('metrics', console.log)
 testAgent.on('process.memory', console.log)
 ```
 
+## Metric Tags
+
+The agent is able to use environment variables as metrics tags. 
+Define a list of environment variables to be used as metric tags:  
+
+```
+# add the values of env. variables USER, PWD as tags to metrics
+export MONITORING_TAGS_FROM_ENV="USER, PWD"
+```
+
+Define static tags in `key:value` format 
+
+```
+export MONITORING_TAGS_FROM_ENV="organisation:sematext, service=api"
+
+```
+
+Both types of tags can be mixed: 
+
+```
+export MONITORING_TAGS_FROM_ENV="organisation:sematext, service:api, USER, PWD"
```

Comment:
Can specify the structure in which the tags need to be, and tell users that it needs to be like that, otherwise it won't work?

Reply:
Described in 2 sections above.